scripts/allowlist/cli/allowlist.js

```diff
@@ -0,0 +1,228 @@
+        }
+    })
+    .command('countVotes <account>', 'count the votes for an account to be added or removed', {
+        account: {
+            description: 'address of the account as a hexadecimal string',
+            type: 'string',
+        }
+    })
+    .command('removeVote <account>', 'remove the vote for an account', {
+        account: {
+            description: 'address of the account as a hexadecimal string',
+            type: 'string',
+        }
+    })
+    .command('getValidators', 'get validators for the last block', {
+    })
+    .command('numAllowedAccounts', 'get the number of allowed accounts for the last block', {
+    })
+    .option('contractAddress', {
+        alias: 'a',
+        demandOption: true,
+        default: "0000000000000000000000000000000000008888",
+        describe: 'private key in hexadecimal format',
+        type: 'string',
+    })
+    .option('privateKey', {
+        alias: 'p',
+        demandOption: true,
+        describe: 'private key in hexadecimal format',
+        type: 'string',
+    })
+    .option('url', {
+        alias: 'u',
+        demandOption: true,
+        default: 'http://localhost:8545',
+        describe: 'URL of the Ethereum client',
+        type: 'string',
+    })
+    .option('chainId', {
+        alias: 'i',
+        demandOption: true,
+        describe: 'chainId of the blockchain',
+        type: 'string',
+    })
+    .help()
+    .alias('help', 'h')
+    .argv;
+
+// console.log(argv);
+
+let provider = new HDWalletProvider({
+    privateKeys:[getHex(argv.privateKey, 64, false, "privateKey")],
+    providerOrUrl:argv.url,
+    chainId:Number(argv.chainId)
+});
+
+web3 = new Web3(provider);
+web3.eth.handleRevert = true;
+
+const node1Account = web3.eth.accounts.privateKeyToAccount("0x70f1384b24df3d2cdaca7974552ec28f055812ca5e4da7a0ccd0ac0f8a4a9b00");
+const node2Account = web3.eth.accounts.privateKeyToAccount("0xad0352cfc09aa0128db4e135fcea276523c400163dcc762a11ecba29d5f0a34a");
+console.log(`Node 1  private key: 0x70f1384b24df3d2cdaca7974552ec28f055812ca5e4da7a0ccd0ac0f8a4a9b00  Account: ${node1Account.address}`)
+console.log(`Node 2  private key: 0xad0352cfc09aa0128db4e135fcea276523c400163dcc762a11ecba29d5f0a34a  Account: ${node2Account.address}`)
+
+
+function prefix0x(need0x, str) {
+    if (need0x && !str.startsWith('0x')) {
+        return "0x" + str;
+    } else if (!need0x && str.startsWith("0x")) {
+        return str.substr(2);
+    }
+    return str;
+}
+
+function getHex(str, len, need0x, name) {
+    var re = XRegExp(`^0x[0-9A-Fa-f]{${len}}$`);
+    if (!re.test(prefix0x(true,str))) {
+        console.log(`ERROR: Invalid hex string for ${name}: ${str}`);
+        if (str.length !== len+2) {
+            console.log(`Expected length is ${len} digits, actual length is ${str.length-2} digits`)
+        }
+        process.exit(-1);
+    }
+    return prefix0x(need0x, str);
+}
+
+function validHexAccount(str) {
+    var re = XRegExp(`^0x[0-9A-Fa-f]{40}$`);
```

Comment:
This would probably better be relegated to [web3.util.isAddress](https://web3js.readthedocs.io/en/v1.5.2/web3-utils.html#isaddress) which not only checks for valid hex, but also implements the EIP55 check hex encoding.

Reply:
Done